bigcache.go

```diff
@@ -12,6 +13,12 @@ const (
 	minimumEntriesInShard = 10 // Minimum number of entries in single shard
 )
 
+// ErrInvalidIteratorState is reported when iterator is in invalid state and cannot be used.
+var ErrInvalidIteratorState = errors.New("Iterator is in invalid state. Use SetNext() to move to next position.")
+
+// ErrCannotRetrieveEntry is reported when entry cannot be retrieved from underlying
+var ErrCannotRetrieveEntry = errors.New("Could not retrieve entry from cache")
+
```

Comment:
Not this line but I assume you are referring to this discussion https://github.com/allegro/bigcache/pull/22#discussion_r83423780